### DISCUSSION TITLE
4.6.22 ARM版本任何方式均无法登录

### DISCUSSION THREAD

**Discussion by SANA1096** [ywmoyue/biliuwp-lite#424] [bug]
### Bug 描述

arm版扫码无反应，验证码登录提示版本过低，x64可正常扫码登陆，已排除手机哔哩哔哩版本问题（IOS,安卓双端测试）

### 操作步骤

```plain text
arm版本验证码可正常获取，但第一次登录显示版本过低，再点击登录需重新获取验证码，无法登录
```

### 期望结果

arm版登录问题得到修复

### 日志

[20231208.log](https://github.com/ywmoyue/biliuwp-lite/files/13614760/20231208.log)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
@ywmoyue 我这里没有arm设备，如果你有的话麻烦看一下吧

**Comment by ywmoyue** [comment]
@gyhmz @SANA1096  麻烦帮忙测试一下这个uwp中的两个二维码能否扫描

[QrCodeTestUwp_1.0.0.0_Test.zip](https://github.com/ywmoyue/biliuwp-lite/files/13622578/QrCodeTestUwp_1.0.0.0_Test.zip)

**Reply by SANA1096** [reply]
第二个二维码可扫描并显示b站账号，第一个二维码扫描无反应

**Reply by GD-Slime** [reply]
这个是arm架构才可以安装的吗? 我这里没有arm设备好像安装不了

**Reply by ywmoyue** [reply]
@GD-Slime 是的，我已经知道问题出在哪了，晚点修复一下

**Comment by ywmoyue** [comment]
@SANA1096 4.6.23 已发布，你验证一下吧

**Reply by SANA1096** [reply]
Arm版扫码还是无效，验证码同样不起作用

**Reply by ywmoyue** [reply]
@SANA1096 视频详情页->右下角更多->手机观看的二维码能扫吗

**Reply by SANA1096** [reply]
还是不可以

**Reply by ywmoyue** [reply]
@SANA1096 试试这个行不行，https://github.com/ywmoyue/TempFileResource/releases/tag/debug_biliuwp_4.6.23.1
还不行的话发一下日志和出来的二维码图片

**Reply by SANA1096** [reply]
辛苦了，已经可以登录了
